client/beta/faq.jsx

```diff
@@ -9,44 +8,40 @@ import QuestionIcon from '../icons/material/ic_help_outline_black_48px.svg'
 
 const questions = [
   {
-    question: 'When will the open beta start?',
+    question: 'What version of BW does ShieldBattery support?',
     answer: [
       <p key="p1">
-        Open beta will start once ShieldBattery is stable and ready for the public. An exact ETA for
-        this is unfortunately impossible to give, but expect to receive regular updates from us as
-        we make progress. Make sure to follow our{' '}
-        <a href="https://twitter.com/shieldbatterybw" target="_blank">
-          Twitter account
-        </a>{' '}
-        to keep up with the latest news and changes.
+        ShieldBattery supports all versions of BW from 1.16.1 and up. The version that is actually
+        run when playing on ShieldBattery is 1.16.1. This means that you can give our application an
+        install path to any version of BW 1.16.1 and above, and things will just work.
       </p>,
       <p key="p2">
-        Even before the open beta starts, we'll be inviting new users to our closed beta
-        periodically. So if you're interested in helping us test the software, finding bugs, and
-        giving feedback before it's available to a wider audience, please{' '}
-        <Link to="/splash">sign up</Link> for an invite.
+        BW:Remaster is not out yet, but to play on it using ShieldBattery, you will have to purchase
+        it from Blizzard.
       </p>,
     ],
   },
   {
-    question: 'What is the status of the closed beta?',
+    question: 'What is the status of the open beta?',
     answer: [
       <p key="p1">
-        The closed beta is currently running, and will last until we feel that the feature set is
-        relatively complete, polished, and bug-free. Our current beta invitees have given us tons of
-        great feedback, and the software is under active development to improve and polish things
-        based on their responses. While the basics of the features we've promised are in place, we
-        are also still actively working on some of the more complex pieces, such as matchmaking.
+        The open beta is currently running, and will last until we feel that the feature set is
+        relatively complete, polished, and bug-free. The point of the closed beta was to test the
+        features we've had in a more controlled fashion and see if we had any leaks, ie. bugs, in
+        our system. Now it's time to open the floodgates, so to speak, as we believe it's time to
+        test our system in a less controlled fashion.
```

Comment:
We've ironed out most of the significant bugs in our base feature set during our closed beta process, and are now ready to test things with a more significant playerbase, as well as build new features that require a larger group.